json-schema-form.d.ts

```diff
@@ -114,20 +155,9 @@ export function createHeadlessForm(
   customConfig?: JSFConfig
 ): HeadlessFormOutput;
 
-type MutationsConfig = {
-  /**
-   * An object with the list of fields to be mutated.
```

Comment:
note: @joaoalmeida-remote @ollyd @dilvane, to clarify, the `modify()` does not mutate the passed JSON Schema. It returns a brand new JSON Schema. That's why I renamed all the "mutation" words to "modify" or "edit".